JobTitle.md

```diff
@@ -16,44 +16,74 @@ _________
+-        taxonomy (required) : classification taxonomy to use; accepted values are onet17, carotenev3, and carotenev3_1. Complete taxonomy lists can be found [here](https://github.com/cbdr/DataScienceAPITaxonomies/tree/master/JobTitle) (restricted to CBReadOnly). Note you could supply more than one taxonomy separated by pipe (see example query below).
 -        title (required if description is empty) : job title
 -        description (required if title is empty) : job description
--        contentLang : the language of the provided text. You should provide this if it is known, as it will improve the accuracy of our translation system. If this parameter is not specified, the service will attempt to detect the language of the text automatically. Accepted language codes are as follows: ar, bg, ca, zhCHS, zhCHT, cs, da, nl, en, et, tl, fi, fr, de, el, ht, he, hi, hu, id, it, ja, kn, ko, lv, lt, ms, mt, no, fa, pl, pt, ro, ru, sk, sl, es, sv, th, tr, uk, ur, vi, cy.
--        outputType : response format, defaults to json; allowable values are json, xml
- 
-Example: https://api.careerbuilder.com/core/classifier/jobtitle?taxonomy=onet15&title=janitor&contentLang=en&outputType=xml
+-        contentLang (use contentLang in version 0.9 and use content_lang in version 1.0) (optional) : the language of the provided text. You should provide this if it is known, as it will improve the accuracy of our translation system. If this parameter is not specified, the service will attempt to detect the language of the text automatically. Accepted language codes are as follows: ar, bg, ca, zhCHS, zhCHT, cs, da, nl, en, et, tl, fi, fr, de, el, ht, he, hi, hu, id, it, ja, kn, ko, lv, lt, ms, mt, no, fa, pl, pt, ro, ru, sk, sl, es, sv, th, tr, uk, ur, vi, cy.
+
+Version 1.0 Example (Highly encouraged to use version 1.0): https://api.careerbuilder.com/core/classifier/jobtitle?title=Janitor&taxonomy=carotenev3.1|Onet17&contentLang=auto
+#Sample Response
+```
+{
+    "data": {
+        "onet17": [
+            {
+                "title": "Janitors and Cleaners, Except Maids and Housekeeping Cleaners",
+                "id": "37-2011.00",
+                "confidence": 90
+            },
+            {
+                "title": "First-Line Supervisors of Housekeeping and Janitorial Workers",
+                "id": "37-1011.00",
+                "confidence": 56
+            }
+        ],
+        "carotenev3.1": [
+            {
+                "title": "Janitor",
+                "id": "37.1",
+                "confidence": 1,
+                "minor_title": "Building Cleaning and Pest Control Workers",
+                "minor_id": "2000"
+            }
+        ]
+    }
+}
+```
+
+
+Version 0.9 Example: https://api.careerbuilder.com/core/classifier/jobtitle?taxonomy=onet17&title=janitor&contentLang=en
```

Comment:
I would just take out the v0.9 example, since it's going away soon anyway. Anybody who's reading the docs must either be writing new code against JTS or modifying some existing code -- either way, they should be looking to upgrade to v1.0 :)